trino/sqlalchemy/dialect.py

```diff
@@ -266,11 +269,15 @@ def has_sequence(self, connection: Connection, sequence_name: str, schema: str =
         """Trino has no support for sequence. Returns False indicate that given sequence does not exists."""
         return False
 
-    def _get_server_version_info(self, connection: Connection) -> Tuple[int, ...]:
+    def _get_server_version_info(self, connection: Connection) -> Any:
         query = "SELECT version()"
-        res = connection.execute(sql.text(query))
-        version = res.scalar()
-        return tuple([version])
+        try:
+            res = connection.execute(sql.text(query))
+            version = res.scalar()
+            return tuple([version])
+        except TrinoUserError as e:
+            logger.debug(f"Failed to get server version: {e.message}")
```

Comment:
Should this be a user-facing warning instead?
I'm not sure what SQLAlchemy uses this version for so maybe it's fine to ignore.

Reply:
This variable is used in each dialect and SQLAlchemy core code doesn't use it as far as I confirmed.
If the library developers didn't implement the method, SQLAlchemy sets None by default, so ignoring or debug log is fine to me.